src/health/health.controller.ts

```diff
@@ -24,13 +24,13 @@ export class HealthController {
       () =>
         this.http.pingCheck(
           'ethconnect-instance',
-          this.tokensService.instanceUrl,
+          this.tokensService.baseUrl,
           basicAuth(this.tokensService.username, this.tokensService.password),
         ),
       () =>
         this.http.pingCheck(
           'ethconnect-contract',
-          this.tokensService.contractInstanceUrl,
```

Comment:
Any reason to have two duplicate ping checks? I assume we can just keep one.

Reply:
Not sure. @calbritt was there a reason for two ping checks?

Reply:
Oh, it was previously for two urls...I'll remove since we now only need one url, `baseUrl`